docs/payments-refunds/create-payments.md

```diff
@@ -84,9 +84,9 @@ Property `state` can have these values:
 - **cancelledBySystem** - no user interactions with payment were made in 5-10 minutes after creation, so our automated job cancelled it.
 - **cancelledByUser** - user cancelled payment inside MobilePay app.
 
-## Get All Payments
+## Get all payments
 
-Theres also a possibility to list all your payments in pages. Multiple searching criteria can be used as query parameters.
+There's also a possibility to list all your payments on pages. Multiple searching criteria can be used as query parameters.
```

Comment:
are you sure it's supposed to be `on` and not `in` like before?

Reply:
should be "in". Fixed.